test/unit/test_actions.py

```diff
@@ -508,6 +515,25 @@ def test_import_states(self, xpatch):
             with open(filename + '.load') as metadata:
                 assert metadata.read() == config_op.state_metadata
 
+    @xtest_patch
+    def test_import_states_skip(self, xpatch):
+        path = '/tmp/data'
+        xpatch.system.ff_patcher.fs.create_dir(path)
+        self.setup_states_data(xpatch.system, state_path=path)
+        LoadSaveConfig(xpatch.system, xpatch.ncs)
+        # import first time into clean environment
+        states = self.invoke_import_state_files(path)
+        assert states is not None
+        # try default import and fail
+        states = self.invoke_import_state_files(path, expected_success=False, skip_existing = True)
+        assert states is None
+        # reimport with skipping already existing states
+        states = self.invoke_import_state_files(path, skip_existing = True)
+        assert states is not None
+        # reimport with overwrite
+        states = self.invoke_import_state_files(path, overwrite = True)
+        assert states is not None
```

Comment:
Is it possible to check more than just `is None` / `is not None`, possibly even if right states were imported, others were skipped?